src/LedgerArk.js

```diff
@@ -109,14 +109,14 @@ LedgerArk.prototype.getAppConfiguration_async = function() {
 LedgerArk.prototype.signPersonalMessage_async = function(path, messageHex) {
 	var splitPath = utils.splitPath(path);
 	var offset = 0;
-	var message = new Buffer(messageHex, 'hex');
+	var message = Buffer.from(messageHex, 'hex');
 	var apdus = [];
 	var response = [];
 	var self = this;
 	while (offset != message.length) {
 		var maxChunkSize = (offset == 0 ? (150 - 1 - splitPath.length * 4 - 4) : 150)
 		var chunkSize = (offset + maxChunkSize > message.length ? message.length - offset : maxChunkSize);
-		var buffer = new Buffer(offset == 0 ? 5 + 1 + splitPath.length * 4 + 4 + chunkSize : 5 + chunkSize);
+		var buffer = Buffer.from(offset == 0 ? 5 + 1 + splitPath.length * 4 + 4 + chunkSize : 5 + chunkSize);
```

Comment:
If you wish to create a new Buffer of a specific length you need to use Buffer.alloc